core/src/session.cpp

```diff
@@ -77,7 +77,7 @@ namespace irods::http
 		}
 
 		if (ec == boost::beast::http::error::body_limit) {
-			log::error("{}: Request constraint error: {}", __func__, ec.message());
+			log::error(*this, "{}: Request constraint error: {}", __func__, ec.message());
```

Comment:
Rename `log` to `logging`.

Just need to replace all instances of this throughout the project.

Reply:
Done.